Mlem.xcodeproj/project.pbxproj

```diff
@@ -847,7 +860,7 @@
 				033F84BA2C2ACB96002E3EDF /* CommentView.swift */,
 				032C32152C36F65500595286 /* ReplyView.swift */,
 				0316CD632C382A6A009EA8EA /* MessageView.swift */,
-				03AF91D82C1A4ED500E56644 /* ImageView.swift */,
+				039D75652C4FC56A004F24C2 /* ImageView */,
```

Comment:
This feels sus

Reply:
`ImageView` is a group I added, and `ImageView.swift` (a file) was moved further up the `pbxproj`. It does look strange lol, but it's fine